src/Rector/Contrib/PHPUnit/SpecificMethod/AssertComparisonToSpecificMethodRector.php

```diff
@@ -132,4 +141,11 @@ private function renameMethod(MethodCall $methodCallNode): void
             'assertFalse' => $falseMethodName,
         ]);
     }
+
+    private function isConstantValue(Node $node): bool
+    {
+        return in_array(get_class($node), [Array_::class, ConstFetch::class], true)
+              || is_subclass_of($node, Scalar::class)
```

Comment:
Using this check we can support all kind of [Scalar Nodes](https://github.com/nikic/PHP-Parser/tree/master/lib/PhpParser/Node/Scalar) :tada:

Reply:
Very nice :1st_place_medal: